datasets/ec-benchmark_dataset_A_1year.txt

```diff
@@ -1,4 +1,4 @@
-Time (YYYY-MM-DD-HH); Significant wave height (m); Zero-up-crossing period (s)
+time (YYYY-MM-DD-HH); significant wave height (m); zero-up-crossing period (s)
```

Comment:
It seems we changed the header of the original file (https://raw.githubusercontent.com/ec-benchmark-organizers/ec-benchmark/master/datasets/A.txt), this PR reverts it to the original header.